vnext/Desktop.DLL/react-native-win32.x64.def

```diff
@@ -50,6 +50,16 @@ EXPORTS
 ?typeName@dynamic@folly@@QEBAPEBDXZ
 ?updateProperties@ShadowNode@react@facebook@@UEAAX$$QEBUdynamic@folly@@@Z
 ?Utf16ToUtf8@Unicode@Common@Microsoft@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@AEBV?$basic_string@_WU?$char_traits@_W@std@@V?$allocator@_W@2@@5@@Z
+?SetRuntimeOptionBool@React@Microsoft@@YAX$$QEAV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@_N@Z
+?SetRuntimeOptionInt@React@Microsoft@@YAX$$QEAV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@H@Z
```

Comment:
Is this specific to a particular native module? If it's part of the general "instance management" API, will we have to transcribe this into the C++/WinRT ABI?

Reply:
It's a generic mechanism to configure arbitrary settings on runtime in any part of the native code base.
According to @acoates-ms, this should be available "out of the box" in RN 0.63, so these functions are transitory.